policybot/pkg/storage/spanner/store.go

```diff
@@ -35,7 +34,9 @@ var scope = log.RegisterScope("spanner", "Spanner abstraction layer", 0)
 func NewStore(context context.Context, database string, gcpCreds []byte) (storage.Store, error) {
 	client, err := spanner.NewClient(context, database, option.WithCredentialsJSON(gcpCreds))
 	if err != nil {
-		return nil, fmt.Errorf("unable to create Spanner client: %v", err)
+		return store{
```

Comment:
It is generally a golang antipattern to return non-nil results when error is not nil.  What was the purpose of this change?

Reply:
If doesn't create this, it will cause type error since store is not a pointer.